nodeJS/express/deployment.md

```diff
@@ -237,6 +237,12 @@ In most cases, you'll be running into errors that thousands of developers have e
 
 There are two stages of the deployment process where you are most likely to encounter problems. These are during deployment and right after.
 
+<div class="lesson-note" markdown="1">
```

Comment:
*Note to self: Look into making warning note boxes more visually distinct from standard note boxes.*

```suggestion
<div class="lesson-note lesson-note--warning" markdown="1">
```